library/alloc/src/string.rs

```diff
@@ -44,6 +44,7 @@
 
 use core::char::{decode_utf16, REPLACEMENT_CHARACTER};
 use core::fmt;
+use core::gc::NoFinalize;
```

Comment:
Should this be `#[unstable(feature = "gc", issue = "none")]`?

Reply:
I believe stability attributes are only required when it's re-exported into libcore or libstd.

Reply:
OK.